DIPs/001/README.md

```diff
@@ -0,0 +1,38 @@
+# Minimum Viable Governance
+This document describes the Minimum Viable Governance of Social Dist0rtion Protocol.
+
+## Goverance
+Governance is the sum of all Dist0rtion Improvement Proposals (DIPs) contained in the master branch of this repository.
+
+### Improve it
+Anyone can open a pull request to this repository to submit a new DIP. If the pull request is approved by the majority of the owners, it is then merged to master, becoming part of the governance.
+
+## Funds
+The funds of the DAO are managed by the multi signature wallet deployed in the Main Ethereum Network at the address `0x527621278422FFf45A66f7086bAeAC770CF12b69`
+
+The multi signature wallet defines a *set of owners* and a *number of required confirmations* to execute transactions.
+
+## Proposals
+Anyone can make a **proposal** by opening an issue in any repository of the Social Dist0rtion Protocol organization on GitHub.
```

Comment:
I wonder if we should use the LeapDAO issue template.

I've been using it since two months now and I think it's really practical and easy to fill out.

This is the template:

```
# Bounty

## Scope
<!-- A list of specific things which should be done to deliver the bounty. These could be seen as requirements to verify/review bounty against -->

-
-
-

## Deliverables
<!-- Artifacts produced as the result of this bounty. Something that could be verified/reviewed. Some examples: updated code, deployment made, blog post published, public event conducted etc -->


## Gain for the project
<!-- How the completion of this bounty helps the project to achieve it is purpose -->


## Roles

bounty gardener: name / share
bounty worker: name / share
bounty reviewer: name / share

```

Reply:
What is the role of the gardener?

Reply:
//cc @johannbarbie (he knows better)

- Gardener: Creates the ticket, can answer questions to the worker, is requesting the payout from the multi signature
- Worker: Does the work
- Reviewer: Reviews the work

Reply:
I like also @vrde version though, it's quite simple, short and straight...

Reply:
I like the LeapDAO template and it can be added in another DIP to extend our governance. I'm reluctant to add it here because it would require also to explain the different roles and I want to keep it simple for now.